roles/splunk/tasks/configure_auditd.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "get {{ splunk_nix_user }} user uid"
+  getent:
+    database: passwd
+    key: "{{ splunk_nix_user }}"
+
+# - debug:
+#     msg: "{{ splunk_nix_user }} user id = {{ ansible_facts.getent_passwd[splunk_nix_user][1] }}"
+
+- name: auditd - set 20-splunk.rules
```

Comment:
Can we have the whole task only run if `auditd` is installed? If it's not installed, the task will fail.

Reply:
I think this is already solved for in `roles/splunk/defaults/main.yml` with the default being `splunk_auditd_configure: false`

This gives the user the ability to enable these tasks, but they aren't enabled by default. Hopefully a user won't enable them if they aren't running an OS that uses auditd.